reana_workflow_controller/rest/utils.py

```diff
@@ -345,19 +346,18 @@ def remove_workflow_workspace(path):
     :param path: Relative path to workspace directory.
     :return: None.
     """
-    reana_fs = fs.open_fs(app.config["SHARED_VOLUME_PATH"])
-    if reana_fs.exists(path):
+    # reana_fs = fs.open_fs(app.config["SHARED_VOLUME_PATH"])
+    if os.path.isdir(path):
         reana_fs.removetree(path)
```

Comment:
Probably `os.rmdir()` could be used here instead of `reana_fs.removetree()`

Reply:
As removetree(path) removes the directory and it's contents and os.rmdir() only removes empty directories, I think shutils.rmtree(path) is a better alternative